integration/helper/namespace.go

```diff
@@ -40,17 +57,19 @@ func NamedTempProject(t *testing.T, client client.Client, name string) *corev1.N
 		Spec:   corev1.NamespaceSpec{},
 		Status: corev1.NamespaceStatus{},
 	}
-	return namedTempCreate(t, client, ns)
+	return tempCreateHelper(t, client, ns)
 }
 
-func namedTempCreate(t *testing.T, client client.Client, namespaceObject corev1.Namespace) *corev1.Namespace {
+func tempCreateHelper(t *testing.T, client client.Client, namespaceObject corev1.Namespace) *corev1.Namespace {
 	t.Helper()
-	namespaceName := namespaceObject.GetNamespace()
+	namespaceName := namespaceObject.GetNamespace() // Will be "" for generateName
```

Comment:
Do you mean `namespaceObject.GetName()` here? I think `namespaceObject.GetNamespace()` is always `""`.